src/components/Request/index.tsx

```diff
@@ -92,18 +87,23 @@ const Request = (props: Props) => {
   }, [props.authorization.isAuth]);
 
   useEffect(() => {
+    const stageList: any[] = [];
+    props.stage?.data.forEach(stageData => {
+      stageList.push(stageData.name);
+    });
     const list: any[] = [];
     props.request.items.forEach(item => {
```

Comment:
when the logic inside useEffect increases in length or complexity, expose it as a local function and call that inside useEffect

Reply:
Separate local function created as per the comments